python_ta/cfg/graph.py

```diff
@@ -335,15 +345,28 @@ def __init__(
         target: CFGBlock,
         edge_label: Optional[Any] = None,
         condition: Optional[NodeNG] = None,
+        negate: Optional[bool] = None,
     ) -> None:
         self.source = source
         self.target = target
         self.label = edge_label
         self.condition = condition
+        self.negate = negate
         self.source.successors.append(self)
         self.target.predecessors.append(self)
         self.z3_constraints = {}
 
+    def get_label(self) -> Optional[Any]:
+        """Return the edge label if specified.
+        If label is None, return edge.negate if sepcified
```

Comment:
there's a typo on the word "specified"; also, I think you should be more precise in how `edge.negate` is used, since we... negate `edge.negate` to generate the label.